mavis/pairing/constants.py

```diff
@@ -10,7 +10,7 @@
 - :term:`split_call_distance`
 """
 DEFAULTS.add(
-    'flanking_call_distance', 10,
+    'flanking_call_distance', 50,
```

Comment:
was this supposed to be changed with the samtools changes?

Reply:
It could have been on its own, but ya I did want to change that default

Reply:
ok that's fine.